x-pack/platform/plugins/shared/stack_connectors/server/connector_types/email/index.ts

```diff
@@ -173,6 +174,7 @@ function validateConfig(
 
 function validateParams(paramsObject: unknown, validatorServices: ValidatorServices) {
   const { configurationUtilities } = validatorServices;
+  const emailValidator = z.array(z.string().max(512)).max(100);
```

Comment:
can we reuse `emailSchema` from `kbn-connector-schemas`?

Reply:
I didn't use them intentionally since they are versioned schema.
We can use it from the `latest` version but a new version would affect here too.
Should we?

Reply:
I think it's ok to use the `latest`